filebeat/tests/system/test_fields.py

```diff
@@ -79,6 +79,6 @@ def test_beat_fields(self):
 
         output = self.read_output()
         doc = output[0]
-        assert doc["beat.name"] == "testShipperName"
-        assert doc["beat.hostname"] == socket.gethostname()
+        assert doc["host.name"] == "testShipperName"
```

Comment:
This should be updated to "host.hostname", no?

Reply:
We already have `host.name` in Beats today since 6.4 and is what is used at the moment everywhere.